src/DDTrace/Configuration.php

```diff
@@ -87,6 +94,55 @@ public function getSamplingRate()
         return $this->floatValue('sampling.rate', 1.0, 0.0, 1.0);
     }
 
+    /**
+     * Returns the sampling rules defined for the current service.
+     * Results are cached so it is perfectly fine to call this method multiple times.
+     * The expected format for sampling rule env variable is:
+     * - example: DD_TRACE_SAMPLING_RULES=[]
+     *        --> sample rate is 100%
+     * - example: DD_TRACE_SAMPLING_RULES=[{"sample_rate": 0.2}]
+     *        --> sample rate is 20%
+     * - example: DD_TRACE_SAMPLING_RULES=[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]
+     *        --> sample rate is 20% except for spans of service starting with 'a' and with name 'b' where rate is 10%
+     *
+     * Note that 'service' and 'name' is optional when when omitted the '*' pattern is assumed.
+     *
+     * @return float
```

Comment:
This actually returns an array, right? Not a float?

Reply:
good catch 👍